napari/components/viewer_model.py

```diff
@@ -396,7 +396,7 @@ def _on_layers_change(self, event):
                 self.dims.set_range(i, r)
         self.events.layers_change()
 
-    def _scroll(self, event):
+    def _scroll(self, viewer, event):
```

Comment:
Why is this change needed?

Reply:
Without the viewer I got errors about the mouse callback expecting 2 arguments but receiving 3. When I looked at an older commit, we were passing the viewer so I added it back in

Reply:
hmm this feels a little funny, i think i see why it was needed, but we might want to improve in a followup - this scroll method shouldn't need to live on the viewer model like this - the mixing of `self` and `viewer` seems like trouble and not something we've needed to do elsewhere.